internal/impl/kube.go

```diff
@@ -234,10 +224,6 @@ func (r *replicaSet) buildDeployment(cfg *KubeConfig) (*appsv1.Deployment, error
 					HostNetwork: cfg.UseHostNetwork,
 				},
 			},
-			Strategy: appsv1.DeploymentStrategy{
-				Type:          "RollingUpdate",
-				RollingUpdate: &appsv1.RollingUpdateDeployment{},
-			},
 			// Number of old ReplicaSets to retain to allow rollback.
 			RevisionHistoryLimit: ptrOf(int32(1)),
```

Comment:
Can we delete `RevisionHistoryLimit` and `MinReadySeconds` now too?

Reply:
Good point about 'RevisionHistoryLimit`. As for `MinReadySeconds` I think it still makes sense, but I guess we can let someone configure it if they really want so we set less knobs in the config.